### DISCUSSION TITLE
Welcome to irritator Discussions!

### DISCUSSION THREAD

**Discussion by quesnel** [vle-forge/irritator#19] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by duboz** [comment]
HI!
happy to subscribe this discussion. Let's start by listing what we'd like to get with irritator in the coming months?
Raph

**Comment by ynedelec3** [comment]
Hi!
I'm not a present Irritator potential user but a goog Guinea Pig.
I succesfully installed it yesterday on Windows 10 64 bit and will share ideas with pleasure as soon as possible -)
Adding a flow element makes app crash as said in issues, a few example models do too.
Yves

**Comment by ynedelec3** [comment]
The present version is much pleasant to use (node number as default name, and shortcut keys which do not mix with node names).
I suggest an idea : It would be nice to see node number on editor graphics too (on header near the type for example)

Output files work well, though  (the issue is closed now) I left in my version a member in observers containing output directory so that files are saved in the right place.

**Reply by quesnel** [reply]
I agree with you. I create an issue.
